it-and-security/lib/macos-check-if-apple-silicon.queries.yml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: query
 spec:
-  name: query_apple_silicon
+  name: Check if Mac has Apple silicon
   query: SELECT 1 FROM system_info WHERE cpu_type = "arm64e";
 # - name: Determine macOS cpu type as Apple Silicon
```

Comment:
I think we can remove these comments if we don't need them? To clean up the file.

Reply:
My hope was we would fix the file format, so we can use a GitOps query. https://github.com/fleetdm/fleet/issues/20747

Reply:
Ohhh, there's a bug rn. 

Makes sense to use the `fleetctl apply` format until we fix the bug.